### ISSUE TITLE
Error running `hdiutil`! Exit code was 1

### ISSUE THREAD

**Issue by arvindr21** [LinusU/node-appdmg#61]
Hello,

I am using node-appdmg for bundling an Electron app. I have configured scripts in my `package.json` to work with the module like

```js
"scripts": {
        "clean": "trash --force build release",
        "build": "npm run clean && electron-packager . 'The Jackal of Javascript' --platform=darwin --arch=x64 --version=0.25.2 --icon ./resources/icon.icns --out ./build --prune --ignore=node_modules/electron-prebuilt --ignore=node_modules/electron-packager --ignore=node_modules/trash --ignore=.git",
        "release": "npm run build && appdmg ./app-dmg.json ./release/app.dmg"
    }
```

and my `app-dmg.json` is

```js
{
    "title": "The Jackal of Javascript",
    "icon": "./resources/icon.icns",
    "background": "./resources/icon.png",
    "icon-size": 80,
    "contents": [
        {
            "x": 448,
            "y": 344,
            "type": "link",
            "path": "/Applications"
        }, {
            "x": 192,
            "y": 344,
            "type": "file",
            "path": "./build/The Jackal of Javascript.app"
        }
    ]
}
```

And when I run `npm run release` I get the below error

```bash
## snipp snipp 
##...
##...
Packaging app for platform darwin x64 using electron v0.25.2
Wrote new app to build/The Jackal of Javascript.app
[ 1/20] Looking for target...                [ OK ]
[ 2/20] Reading JSON Specification...        [ OK ]
[ 3/20] Parsing JSON Specification...        [ OK ]
[ 4/20] Validating JSON Specification...     [ OK ]
[ 5/20] Looking for files...                 [ OK ]
[ 6/20] Calculating size of image...         [ OK ]
[ 7/20] Creating temporary image...          [ OK ]
[ 8/20] Mounting temporary image...          [ OK ]
[ 9/20] Making hidden background folder...   [ OK ]
[10/20] Copying background...                [ OK ]
[11/20] Reading background dimensions...     [ OK ]
[12/20] Copying icon...                      [ OK ]
[13/20] Setting icon...                      [ OK ]
[14/20] Creating links...                    [ OK ]
[15/20] Copying files...                     [ OK ]
[16/20] Making all the visuals...            [ OK ]
[17/20] Blessing image...                    [ OK ]
[18/20] Unmounting temporary image...        [ OK ]
[19/20] Finalizing image...                  [FAIL]
[20/20] Removing temporary image...          [ OK ]

Error: Error running `hdiutil`! Exit code was 1

npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "run" "release"
npm ERR! node v0.12.2
npm ERR! npm  v2.10.0
npm ERR! code ELIFECYCLE
npm ERR! wordpress-offline-viewer@0.0.0 release: `npm run build && appdmg ./app-dmg.json ./release/app.dmg`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the wordpress-offline-viewer@0.0.0 release script 'npm run build && appdmg ./app-dmg.json ./release/app.dmg'.
npm ERR! This is most likely a problem with the wordpress-offline-viewer package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run build && appdmg ./app-dmg.json ./release/app.dmg
npm ERR! You can get their info via:
npm ERR!     npm owner ls wordpress-offline-viewer
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
```

I tried
* Restarting my machine
* creating  `.app` and `.dmg` without spaces in the title
* using icons from this repo in my dmg
no luck,

Any help is appreciated.

Thanks.

**Comment by arvindr21** [comment]
Updating the script as below works

```js
// snipp snipp
"release": "npm run build && appdmg ./app-dmg.json 'The Javascript of Javascript.dmg'"
// snipp snipp
```
for some reason, I am not able to write into the `release` folder. Any thoughts?

**Comment by arvindr21** [comment]
Hello again,

Adding one more issue..

No background image present in the `dmg`. I have tested with BG image from the help folder and also a modified version of the same. Same `.json` file as above.

The generated `dmg` looks as below.

![screen shot 2015-05-24 at 10 29 02 am](https://cloud.githubusercontent.com/assets/2562992/7786747/31098a58-01ff-11e5-9f94-4a422e51df41.png)

The background Image I have used is 
![bg](https://cloud.githubusercontent.com/assets/2562992/7786749/42871e26-01ff-11e5-93b0-68851d5d1b5e.png)

Any clue as to what I am doing wrong?

Thanks.
